src/App.js

```diff
@@ -1,14 +1,21 @@
 import './App.css';
 import {Checklist} from './Checklist.js'
 import {AddTaskPage} from "./AddTaskPage";
-import {useEffect, useState} from "react";
+import React, {useEffect, useState} from "react";
 import {IncompleteTasksOnly} from "./IncompleteTasksOnly";
 import {generateUniqueID} from "web-vitals/dist/modules/lib/generateUniqueID";
 import {ModifyTaskPage} from "./ModifyTaskPage";
+import {useCollection} from "react-firebase-hooks/firestore";
+import firebase from "firebase/compat";
 
 export function App(props) {
-
-    const [data, setData] = useState(props.initialData);
+    const [sortValue, setSortValue] = useState("title");
+    const [value, loading, error] = useCollection(props.collection.orderBy(sortValue, "asc"));
```

Comment:
You want high priority tasks to come at the *end* of the list?

Reply:
They come at the start of the list currently.